program/src/distribute_base_ncn_reward_route.rs

```diff
@@ -0,0 +1,78 @@
+use jito_bytemuck::AccountDeserialize;
+use jito_restaking_core::{config::Config, ncn::Ncn, operator::Operator};
+use jito_tip_router_core::{
+    base_reward_router::BaseRewardRouter, error::TipRouterError, loaders::load_ncn_epoch,
+    ncn_config::NcnConfig, ncn_fee_group::NcnFeeGroup, ncn_reward_router::NcnRewardRouter,
+};
+use solana_program::{
+    account_info::AccountInfo, clock::Clock, entrypoint::ProgramResult, msg,
+    program_error::ProgramError, pubkey::Pubkey, sysvar::Sysvar,
+};
+
+/// Can be backfilled for previous epochs
+pub fn process_distribute_base_ncn_reward_route(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    ncn_fee_group: u8,
+    first_slot_of_ncn_epoch: Option<u64>,
```

Comment:
IMO makes more sense just pass in the epoch itself here as a non optional value

Reply:
Will change in refactor